### ISSUE TITLE
Issues with installation using pipenv

### ISSUE THREAD

**Issue by Sarumyan** [HBNetwork/python-decouple#169]
There is a problem with installing python-decouple using pipenv.

When I run `pipenv install python-decouple` and then try `from decouple import config`, I get an error:
`ImportError: cannot import name 'config' from 'decouple' (/home/dmor/.local/share/virtualenvs/decouple_tests-fWNCkI-j/lib/python3.10/site-packages/decouple/__init__.py)`
And `pip freeze` shows me:
```
decouple==0.0.7
python-decouple==3.8
```

And if I run instead `pip install python-decouple` everything works fine.
And `pip freeze` shows:
```
python-decouple==3.8
```

So basically pipenv also installs decouple itself which breaks the python-decouple.
As pipenv is a quite popular tool, I think this is the issue worth addressing.

**Comment by henriquebastos** [comment]
`decouple` and `python-decouple` are 2 different libraries.
